docs/UserGuide.md

```diff
@@ -557,21 +626,24 @@ Examples:
 * `paid 1 1 amt/70` The 1st student has paid `$70.00` for his or her 1st lesson.
 * `paid 3 2 amt/480.50` The 3rd student has paid `$480.50` for his or her 2nd lesson.
 
-#### Behaviours of the Fees Calculator
+#### Behaviour of the Fees Calculator
 
 TAB will automatically update your lesson's outstanding fees once the lesson has ended using Fees Calculator feature. 
 The Fees Calculator will account for cancelled dates and ensure that lesson fees on these dates will not be added.
 
-However, the Fees Calculator will not account for any changes to lessons that have passed. Such cases include:
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+The Fees Calculator will not account for any changes to lessons that have passed.
+</div>
 
+Such cases to take note of include:
 * **Lesson rates increment.** In the event that you want to increase your lesson rates, the current outstanding fees will not change according to the newly edited lesson rates.
-* **Incorrect lesson rates entry.** Similarly, in the event that you have entered your lesson rates incorrectly and only realised it after your lesson has passed, the current outstanding fees will not change according to 
-the newly edited lesson rates.
+* **Incorrect lesson rates entry.** Similarly, in the event that you have entered your lesson rates incorrectly and only realised it after your lesson has passed, the current outstanding fees will not change according to
+  the newly edited lesson rates.
 * **Cancelling or uncancelling a date in the past.** In the event that you did not cancel your lesson and the fees for that particular cancelled lesson has been added to outstanding fees, the Fees Calculator will not deduct
-the fees of the cancelled lesson for you. Same for uncancelling a lesson that has passed, the fees will not be added back for you.
+  the fees of the cancelled lesson for you. Same for uncancelling a lesson that has passed, the fees will not be added back for you.
 * **Shifting the end date of a recurring lesson.** In the event that the end date of the lesson is shifted to an earlier date and lessons after that new end date have already passed, the outstanding fees will not change.
 * **Shifting the start date of a recurring lesson.** In the event that the start date of the lesson is shifted to an earlier date and lessons between the edited start date and original start date have passed,
-the fees of these lessons will not be deducted for you. Same for shifting start date to a later date after the original start date has passed and fees have been updated prior.
+  the fees of these lessons will not be deducted for you. Same for shifting start date to a later date after the original start date has passed and fees have been updated prior.
```

Comment:
The whole part about fees wasn't done by me, I think the IDE marked it as green because it detected some format changes when I put it into the caution block and then took it out again.

Reply:
refer to #314